changelog.d/20250204_153946_PL-133028_mysql-slowlog-alloy_scriv.md

```diff
@@ -0,0 +1,7 @@
+### Impact
+
+-
+
+### NixOS XX.XX platform
+
+- Add configuration for alloy that exports MySQL's slow logs to loki if configured for the resource group
```

Comment:
I haven't been writing changelogs for Loki-related platform features given that the integration is still considered WIP. But if you do leave it in, adding a ticket number would be nice.